app/services/doc_auth/mock/result_response.rb

```diff
@@ -195,9 +211,15 @@ def create_response_info(
           },
           alert_failure_count: failed&.count.to_i,
           image_metrics: merged_image_metrics,
-          portrait_match_results: { FaceMatchResult: liveness_result },
+          liveness_enabled: liveness_enabled,
           classification_info: classification_info,
-        }
+        }.compact
+
+        if @selfie_check_performed
+          true_id_mock_response[:portrait_match_results] = portrait_match_results
+        end
```

Comment:
since we added the `#compact` already I think we can do this inline instead of having to save a reference to the hash as a variable
```suggestion
          portrait_match_results: @selfie_check_performed ? portrait_match_results : nil,
        }.compact
```

Reply:
Oh nice!